assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -56,14 +65,55 @@ function useUploader(refreshLibrary = true) {
     [maxUpload]
   );
 
+  const validErrorMessage = createInterpolateElement(
+    sprintf(
+      __('Please choose only <b>%s</b> to upload.', 'web-stories'),
```

Comment:
This needs a comment to explain the placeholder. See https://github.com/google/web-stories-wp/blob/9df43e73705c77b718d2fee9f5fed4ff779370b1/assets/src/edit-story/components/uploadDropTarget/message.js#L80

Reply:
Added